node/src/options.rs

```diff
@@ -86,6 +80,50 @@ pub struct RunOptions {
     pub rpc_addr: Option<SocketAddr>,
 }
 
+/// Run the regtest node.
+#[derive(Args, Debug)]
+pub struct RegtestOptions {
+    #[clap(flatten)]
+    pub run_options: RunOptions,
+    #[clap(flatten)]
+    pub chain_config: ChainConfigOptions,
+}
+
+#[derive(Args, Debug)]
+pub struct ChainConfigOptions {
```

Comment:
One thing you can add here is the block subsidy. This is one thing we need to test in regtests. The idea is that it should be possible to submit block subsidy values in ChainConfig. There's a translation mechanism for that, where strings can be converted to a subsidy schedule. You don't have to do this in this PR, but I don't see why not. It's your call whether you want to make a separate PR for that if you find it more complex than this.

Reply:
Btw, there's more stuff. This is just the beginning. NetUpdates, checkpoints and more should all be passed here. But they're still not parsable as strings. That comes later.